File: src/main/java/net/hectus/neobb/turn/person_game/categorization/WarpCategory.java
package net.hectus.neobb.turn.person_game.categorization;

import net.kyori.adventure.text.format.NamedTextColor;
import net.kyori.adventure.text.format.TextColor;
import org.bukkit.Material;

public interface WarpCategory extends Category {
    @Override
    default TextColor categoryColor() {
        return NamedTextColor.BLUE;
    }

    @Override
    default Material categoryItem() {
        return Material.END_ROD;
    }

    @Override
    default String categoryName() {
        return "warp";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/categorization/WarpCategory.kt
package net.hectus.neobb.modes.turn.person_game.categorization

import net.kyori.adventure.text.format.NamedTextColor
import net.kyori.adventure.text.format.TextColor
import org.bukkit.Material

interface WarpCategory : Category {
    override val categoryColor: TextColor get() = NamedTextColor.BLUE

    override val categoryItem: Material get() = Material.END_ROD
    override val categoryName: String get() = "warp"
}
